core/cold-wallet/communication/masternode-communication.ts

```diff
@@ -0,0 +1,15 @@
+// import { BaseSerialCommunication } from '../../shared/communication/base-serial-communication';
+
+import { BaseServerCommunication } from '../../shared/communication/base-server-communication';
+
+// export class MasternodeCommunication extends BaseSerialCommunication {
+//   getPath(): string {
+//     return '/dev/serial0';
+//   }
+// }
+
+export class MasternodeCommunication extends BaseServerCommunication {
+  listenOnPort(): number {
+    return 9000;
+  }
+}
```

Comment:
Why don't you use constructor arguments?

Reply:
I had constructor arguments in the beginning but this would render `MasternodeCommunication` pointless. Therefore I have gone with an `abstract` class and that little function.

We can switch to constructor argument in the next PR, because I am anyways working on that class again.

In the end it doesn't really matter, as this is just for local development purposes anyways

Reply:
Ok, was just curious 👍

Reply:
perfectly fine :D
Do you see any advantages by having a constructor argument instead of abstract function? I mean there will just be one of this object...

Reply:
```ts
export class MasternodeCommunication extends BaseServerCommunication {
   super(9000);
}
```
is a bit shorter, but also less readable than
```ts
export class MasternodeCommunication extends BaseServerCommunication {
  listenOnPort(): number {
    return 9000;
  }
}
```

Maybe an abstract property would be simpler 🤔